tsconfig.json

```diff
@@ -17,134 +17,137 @@
 		"./typings/tsd.d.ts"
 	],
 	"files": [
-		"./src/DateObject.ts",
```

Comment:
The latest builds of TypeScript/atom-typescript have regenerated the file without the relatives paths... we can keep ignoring it, or update it at some point.  It wouldn't have to come in with this commit, but I just got tired of pretending the file didn't change.